docs/policies.rst

```diff
@@ -456,9 +466,9 @@ Default Actions for Affirmation and Rephrasing
 """"""""""""""""""""""""""""""""""""""""""""""
 
 Rasa Core provides the default implementations
-``action_default_ask_affirmation`` and ``action_default_ask_rephrase``
-which are triggered when the bot should ask the user to affirm
-or to rephrase their intent.
+``action_default_ask_affirmation``, ``action_default_ask_rephrase``, and
+``action_default_fallback`` which are triggered when the bot should ask the
+user to affirm their intent, to rephrase it, or as ultimate fallback.
```

Comment:
I applied your recommendation but I prefered to revert the change and omit ` ``action_default_fallback`` is called if the rephrasing was not recognized.`  since this should be clear from the table above and `fallback_nlu_action_name: "action_default_fallback"`. Is that ok?

Reply:
I approve